ui/src/dashboards/selectors/index.ts

```diff
@@ -16,6 +27,12 @@ export const getViewStatus = (state: AppState, id: string): RemoteDataState => {
   return get(state, `views.views.${id}.status`, RemoteDataState.Loading)
 }
 
+export const getTimeRangeByDashboardID = (
```

Comment:
probably a good idea to write a test for this too since so many parts of the UI now depend on this